### PR TITLE
Feature/aws deployment

### PR DESCRIPTION
**Pull request by cherrythia**
PR for AWS deployment

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
+package config
+
+import (
+	"fmt"
+	"github.com/spf13/viper"
+)
+
+var (
```
Comment (sweiyang) [inline config/config.go:8]: normal for unchanging variables we use "const"

File: controller/provider/provider_payment_controller.go [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+var (
+	ProviderPaymentControllerObj ProviderPaymentController
+)
+
+type ProviderPaymentControllerImpl struct {
+	stripeKey string
+}
+
+func (p ProviderPaymentControllerImpl) GetProviderPaymentInfo(providerId uint, billingPeriod dto.ProviderBillingPeriod) ([]dto.ProviderPayment, error) {
+	providerPayment, err := dao.Db.GetProviderPaymentEntry(providerId, billingPeriod)
```
Comment (sweiyang) [inline controller/provider/provider_payment_controller.go:28]: you could have just 

return dao.Db.GetProviderPaymentEntry(providerId, billingPeriod)

File: controller/provider/provider_payment_controller.go [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+	return dao.Db.DeleteProviderPaymentEntry(id, billingPeriod)
+}
+
+func (p ProviderPaymentControllerImpl) CompleteProviderPayment(providerPayment *dto.ProviderPayment) error {
+	it, err := p.GetProviderPaymentInfo(providerPayment.ProviderId, providerPayment.ProviderBillingPeriod)
+	if err != nil {
+		return err
+	}
+
+	if len(it) > 0 {
```
Comment (sweiyang) [inline controller/provider/provider_payment_controller.go:49]: could we abstract this function to dto? 
payment.SetComplet() error 

return errors with the validation beflow

File: controller/user/user_payment_controller.go [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+import (
+	"ev-payment-service/config"
+	"ev-payment-service/dao"
+	"ev-payment-service/dto"
+	helper "ev-payment-service/helper"
+	"fmt"
+	"github.com/sirupsen/logrus"
+)
+
+type UserPaymentController interface {
```
Comment (sweiyang) [inline controller/user/user_payment_controller.go:12]: this is the tricky portion hahah, actually we are doing the same thing. so actually payment dto is an interface. 

then extends providerPayment and userPayment. its ok no need change for now

Comment (cherrythia) [reply controller/user/user_payment_controller.go:12]: yes I agree, i realised this later on after userPayment is done. Lazy to refactor.

**Pull request merged**